swagger.json

```diff
@@ -4529,30 +4529,30 @@
                 },
                 "destination_name": {
                     "type": "string",
-                    "description": "The name of the job destination.",
+                    "description": "The name of the job destination. Optional if a destination address ID is provided. If the destination name is provided, it will take precedence over the name of the address book entry.",
                     "example": "ACME Inc. Philadelphia HQ"
                 },
                 "destination_address": {
                     "type": "string",
-                    "description": "The address of the job destination, as it would be recognized if provided to maps.google.com",
+                    "description": "The address of the job destination, as it would be recognized if provided to maps.google.com. Optional if a valid destination address ID is provided.",
                     "example": "123 Main St, Philadelphia, PA 19106"
                 },
                 "destination_address_id": {
                     "type": "integer",
                     "format": "int64",
-                    "description": "ID of the job destination associated with an address book entry.",
+                    "description": "ID of the job destination associated with an address book entry. Optional if valid values are provided for destination name, address, latitude and longitude. If a valid destination address ID is provided, address/latitude/longitude will be fetched from the address book, and name will be fetched from the address book only if it is omitted in the request body.",
```

Comment:
nit: Maybe do not use word "fetched"? Honestly we should standardize on copy within the API. Maybe "If a valid destination address ID is provided, address/latitude/longitude will be used from the address book entry. Name of the address book entry will only be used if destination_name is not provided."